development/rails/database_setup/README.md

```diff
@@ -4,45 +4,54 @@
 
 ### Creating & Migrating the Database
 
+In order to use structure.sql, in `config/application.rb`
+
+```
```

Comment:
Can you update this so GitHub gives it nice syntax highlighting? I.e. add `ruby` after the triple "`"